### PR TITLE
Use common stylesheet

### PR DESCRIPTION
**Pull request by shammamah**
## Description of changes 
* Change apps to use the `general-app-page` stylesheet. 

- [x] Alignment Viewer 
- [x] Circos 
- [x] Clustergram
- [x] Ideogram 
- [x] Manhattan Plot
- [x] Needle Plot
- [x] Onco Print
- [x] Sequence Viewer 
- [x] Speck 
- [x] Volcano Plot

## Before merging
- [x] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [x] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [x] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

File: assets/alignmentchart-style.css [hunk 1]
```diff
@@ -1,23 +1,5 @@
...
-    height: 0px;
-    margin-right: 500px;
-    background-color: #f2f7f7;
-}
 #alignment-body ::-webkit-scrollbar-thumb {
-    width: 2px;
-    background-color: #34ad99;
-    border-radius: 15px;
-    padding-right: 5px;
+    background-color: #34ad99 !important;
```
Comment (mkcor) [inline assets/alignmentchart-style.css]:
```suggestion
    background-color: #34AD99 !important;
```
(so it looks the same as line 25)

File: assets/alignmentchart-style.css [hunk 2]
```diff
@@ -107,49 +59,32 @@
...
 h3.alignment-settings-section {
     text-align: center;
-    margin-bottom: 20px;
+    margin-bottom: 15px;
+    font-size: 16pt;
 }
 
-h4 {
-    text-align: center;
+#alignment-colorscale-dropdown, #alignment-overview-dropdown, #alignment-conservationcolorscale-dropdown, #alignment-conservationmethod-dropdown, #alignment-gapcolor-dropdown {
```
Comment (mkcor) [inline assets/alignmentchart-style.css]: Is it possible to break this line? I'm not familiar enough with CSS syntax to make a suggestion.

File: assets/ideogram-style.css [hunk 3]
```diff
@@ -1,70 +1,18 @@
...
-    overflow: auto;
+    background-color: #FDFCFF !important;
+    color: #303030 !important;
 }
 #ideogram-body ::-webkit-scrollbar {
-    width: 4px;
-    height: 0px;
-    margin-right: 500px;
-    background-color: #fdfcff;
+    background-color: #fdfcff !important;
```
Comment (mkcor) [inline assets/ideogram-style.css]:
```suggestion
    background-color: #FDFCFF !important;
```

File: assets/sequence-viewer-style.css [hunk 4]
```diff
@@ -283,16 +236,14 @@ input#coverage-color, input#coverage-bg-color, input#coverage-tooltip {
...
 }
-
-#seq-view-body button:hover {
-    border: solid 1px #C50063;
+#seq-view-body .control-upload {
+    border-color: #C50063;
+}
+#seq-view-body .control-download:hover {
     color: #C50063;
+    border-color: #C50063
```
Comment (mkcor) [inline assets/sequence-viewer-style.css]:
```suggestion
    border-color: #C50063;
```

File: assets/alignmentchart-style.css (hunk 2 above)
Comment (shammamah) [reply assets/alignmentchart-style.css]: Done in e492ed6!

Comment (mkcor) [reply assets/alignmentchart-style.css]: @shammamah I guess you meant b6388d4? It looks like e492ed6 shouldn't be in (it's an empty commit). Do you mind removing it by rebasing interactively?

**Pull request merged**